.travis.yml

```diff
@@ -12,6 +12,10 @@ addons:
       - libboost-all-dev
       - libxml2-dev
       - libsnappy-dev
+stages:
```

Comment:
I have a feeling that we can avoid stages (they are slow in case of long setup like we have here) and implement it as a regular `deploy` in our Python tasks